contracts/BosonRouter.sol

```diff
@@ -1192,18 +1192,18 @@ contract BosonRouter is
      * @param _tokenIdSupply    ID of the supply token
      * @param _gateAddress address of a gate contract that will handle the interaction between the BosonRouter contract and the non-transferrable NFT,
      * ownership of which is a condition for committing to redeem a voucher in the voucher set created by this function.
-     * @param _nftTokenId Id of the NFT (ERC115NonTransferrable) token, ownership of which is a condition for committing to redeem a voucher
+     * @param _conditionalTokenId Id of the conditional token, ownership of which is a condition for committing to redeem a voucher
      * in the voucher set created by this function.
      */
-    function finalizeConditionalOrder(uint256 _tokenIdSupply, address _gateAddress, uint256 _nftTokenId) internal {
+    function finalizeConditionalOrder(uint256 _tokenIdSupply, address _gateAddress, uint256 _conditionalTokenId) internal {
         voucherSetToGateContract[_tokenIdSupply] = _gateAddress;
 
         emit LogConditionalOrderCreated(_tokenIdSupply, _gateAddress);
```

Comment:
Agreed. But it won't always be present.

- For ERC20, the contract is the token, so it would never have a tokenId registered
- For ERC1155, it needs a tokenId always.
- For ERC721, it's interesting because you could target
  - the owner of a **specific** NFT (Avastar number 153) 
  - holders of **any** NFT on the conditional token contract
  
 I am only testing the use of a conditional token ID on the Quest and ERC1155.

Reply:
The code change looks fine, but it looks like the test cases that check LOG_CONDITIONAL_ORDER_CREATED in 12_conditional_commit.ts weren't changed to check for the conditional token Id.

Reply:
The code changes were made, but it looks like the LOG_CONDITIONAL_ORDER_CREATED assertions in 12_conditional_commit.ts weren't updated to check the conditional token Id